packages/docs/src/components/Pricing/pricingBusinessData.tsx

```diff
@@ -0,0 +1,65 @@
+import { PricingPlanCardProps } from "./PricingPlansList";
+
+export const pricingBusinessData: PricingPlanCardProps[] = [
+  {
+    plan: "TEAM",
+    price: { monthly: 75, yearly: 750 },
+    yearlyFullPrice: 900,
+    label: "For small and medium teams",
+    buttonLabel: "Buy your licenses",
+    stylingFilled: true,
+    featuresAll: [
+      { label: "Element inspector", info: "" },
+      { label: "Debugging and logging", info: "" },
+      {
+        label: "Dev tools",
+        info: "Outline render, JavaScript CPU profiler, Redux DevTools integration, React Query devtools plugin, React Profiler integration",
+      },
+      { label: "Isolated components preview", info: "" },
+      { label: "Screenshots & screen recording", info: "" },
+      { label: "Replays", info: "" },
+      { label: "Extended device emulator", info: "Phones and tablets" },
```

Comment:
nit: when you make `info` an optional parameter you could remove a considerable amount of boilerplate code
```suggestion
      { label: "Element inspector" },
      { label: "Debugging and logging" },
      {
        label: "Dev tools",
        info: "Outline render, JavaScript CPU profiler, Redux DevTools integration, React Query devtools plugin, React Profiler integration",
      },
      { label: "Isolated components preview" },
      { label: "Screenshots & screen recording" },
      { label: "Replays" },
      { label: "Extended device emulator", info: "Phones and tablets" },
```